core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -643,6 +664,9 @@ default Iterable<Entry<String,String>> getProperties(String tableName)
   Map<String,String> getConfiguration(String tableName)
       throws AccumuloException, TableNotFoundException;
 
+  Map<String,String> getTableSpecificConfiguration(String tableName)
```

Comment:
How is this different than `getConfiguration` above?

Reply:
I just ran `TableOperationsIT.createTable` and modified it to print out the returned properties. You weren't joking. It looks like `getConfiguration(String)` was added in 2.1.0.  I wonder if that method should be renamed. We should get some other opinions here. @milleruntime @ctubbsii @EdColeman

Reply:
@dlmarion - Yeah I figured based on that it would make a lot more sense to filter the returned properties from the server vs trying to do it later on the client side and sending all of that useless stuff hence the new method/call. I'd think either renaming the method or having a flag/argument to filter what is returned makes sense. (ie return all, return just table props, etc).

Reply:
If this is returning the properties from ZooKeeper that are key by id, then the naming would be getTableProperties and getNamespaceProperties - to me configuration should imply the configuration hierarchy, that hierarchy if build from properties and possibly other sources (say configuration files,...)